### ISSUE TITLE
yaml selector rely on another selector in conda_build_config.yaml? If not, mps as an ecosystem-wide selector? 

### ISSUE THREAD

**Issue by ngam** [conda-forge/conda-forge-pinning-feedstock#2926]
Can I make a selector inside conda_build_config.yaml rely on another selector like below? I want to make a new selector called "osx_impl" and then use it to set a target.

copied from https://github.com/conda-forge/conda-forge.github.io/issues/1763

xref: https://github.com/conda-forge/pytorch-cpu-feedstock/pull/106

conda_build_config.yaml: 

```yaml
osx_impl:   # [osx and x86_64]
   - mps    # [osx and x86_64]
   - nomps  # [osx and x86_64]

 MACOSX_DEPLOYMENT_TARGET:  # [osx_impl == "mps"]
   - "12.3"                 # [osx_impl == "mps"]
 MACOSX_SDK_VERSION:        # [osx_impl == "mps"]
   - "12.3"                 # [osx_impl == "mps"]
```

**Comment by hmaarrfk** [comment]
I don't think this is possible with the syntax you purposed. 

You might have to use zip keys or whatnot.

**Comment by ngam** [comment]
> I don't think this is possible with the syntax you purposed.
> 
> You might have to use zip keys or whatnot.

Any tips/examples would be appreciated :)

**Comment by ngam** [comment]
resolved in [6f578fa](https://github.com/conda-forge/pytorch-cpu-feedstock/pull/106/commits/6f578fa2f824364423e1d90588c67b5749e8862b)
